### PR TITLE
bind tileController during initialization AudioVideoController

### PR DESCRIPTION
**Pull request by lzongkun**
**Issue #:**
Context:
tileController is required for the VideoPriorityBasedPolicy in order to pause/unpause video tiles based on calculation.
Instead of depending on users to call bindToTileController() which they often forget, we can bind the tileController during the initialization of AudioVideoController, where we init the meetingSession.

**Description of changes:**
bind tileController during initialization of DefaultAudioVideoController for VideoPriorityBasedPolicy
**Testing**

1. Have you successfully run `npm run build:release` locally? Yes
2. How did you test these changes? Tested with demo app and verified that tiles being paused under network constraint.
3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it? Yes, used meetingDemoV2
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? No.
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? No.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
```
Comment (ltrung) [inline CHANGELOG.md:10]: nit: Move to the next Added section.

File: src/videodownlinkbandwidthpolicy/VideoPriorityBasedPolicy.ts [hunk 2]
```diff
@@ -124,6 +124,7 @@ export default class VideoPriorityBasedPolicy implements VideoDownlinkBandwidthP
 
   bindToTileController(tileController: VideoTileController): void {
     this.tileController = tileController;
+    this.logger.info('tileController binded');
```
Comment (ltrung) [inline src/videodownlinkbandwidthpolicy/VideoPriorityBasedPolicy.ts:127]: nit: Should be bound.

File: demos/browser/app/meetingV2/meetingV2.ts [hunk 3]
```diff
@@ -1386,15 +1387,17 @@ export class DemoMeetingApp
       configuration.attendeePresenceTimeoutMs = Number(timeoutMs);
     }
     configuration.enableSimulcastForUnifiedPlanChromiumBasedBrowsers = this.enableSimulcast;
+    if (this.usePriorityBasedDownlinkPolicy) {
+      this.priorityBasedDownlinkPolicy = new VideoPriorityBasedPolicy(this.meetingLogger);
+      configuration.videoDownlinkBandwidthPolicy = this.priorityBasedDownlinkPolicy;
+      this.priorityBasedDownlinkPolicy.addObserver(this);
```
Comment (ltrung) [inline demos/browser/app/meetingV2/meetingV2.ts:1393]: Is this change related?

Comment (lzongkun) [reply demos/browser/app/meetingV2/meetingV2.ts:1393]: > Is this change related?

Yes, this is to make sure we set the downlink policy in meetingSessionConfiguration before we use it to init the meetingSession, which is what we expect our customers to do.

**Review by ltrung — APPROVED** [review]
